api/common.js

```diff
@@ -28,6 +28,7 @@ exports.disconnectRedis = ()=>{
 let amqp_conn;
 function get_amqp_connection(cb) {
     if(amqp_conn) return cb(null, amqp_conn); //already connected
+    console.debug("connecting to amqp");
```

Comment:
Wouldn't it be confusing that we see this message twice now? Maybe remove it?

Reply:
Yeah I can remove it. This was to check the code was getting there (connecting vs connected).